src/innsyn/LoadingResourcesFailedAlert.tsx

```diff
@@ -1,57 +1,73 @@
-import {Alert, BodyShort} from "@navikt/ds-react";
+import {Alert} from "@navikt/ds-react";
 import React, {useEffect} from "react";
-import {REST_STATUS} from "../utils/restUtils";
 import {logServerfeil} from "../utils/amplitude";
-import {useSelector} from "react-redux";
-import {InnsynAppState} from "../redux/reduxTypes";
+import {
+    useGetDokumentasjonkrav,
+    useGetOppgaver,
+    useGetVilkar,
+} from "../generated/oppgave-controller/oppgave-controller";
+import {useHentSoknadsStatus} from "../generated/soknads-status-controller/soknads-status-controller";
+import {useHentHendelser} from "../generated/hendelse-controller/hendelse-controller";
+import {useHentVedlegg} from "../generated/vedlegg-controller/vedlegg-controller";
+import {FormattedMessage} from "react-intl";
+import styled from "styled-components";
 
-const leserData = (restStatus: REST_STATUS): boolean => {
-    return restStatus === REST_STATUS.INITIALISERT || restStatus === REST_STATUS.PENDING;
-};
+const StyledWrapper = styled.div`
+    position: sticky;
+    top: 0;
+    z-index: 1;
+`;
 
 export const LoadingResourcesFailedAlert = (props: {
+    fiksDigisosId: string;
     loadingResourcesFailed: boolean;
     setLoadingResourcesFailed: (loadingResourcesFailed: boolean) => void;
 }) => {
-    const {saksStatus, oppgaver, dokumentasjonkrav, vilkar, soknadsStatus, hendelser, vedlegg} = useSelector(
-        (state: InnsynAppState) => state.innsynsdata.restStatus
-    );
+    const {isError: soknadsStatusHasError} = useHentSoknadsStatus(props.fiksDigisosId);
+    const {isError: oppgaverHasError} = useGetOppgaver(props.fiksDigisosId);
+    const {isError: vilkarHasError} = useGetVilkar(props.fiksDigisosId);
+    const {isError: dokumentasjonkravHasError} = useGetDokumentasjonkrav(props.fiksDigisosId);
+    const {isError: hendelserHasError} = useHentHendelser(props.fiksDigisosId);
+    const {isError: vedleggHasError} = useHentVedlegg(props.fiksDigisosId);
```

Comment:
Kan vi prøve å flytte datauthenting/loading/feilmelding til der hvor dataen brukes? Så i stedet for at man må hente ned hele sulamitten her, så kan `<Oppgaver>`-komponenten ha ansvar for å vise "sin" nettverkstilstand? Det er for øvrig det jeg prøver å gjøre i min refaktorering, selv om det har sporet litt av.